tests/test_e2e.py

```diff
@@ -102,15 +96,15 @@ def test_generate_election_diff_res():
     pp = PreferenceProfile(ballots=[b1, b2, b3, b4, b5, b6])
 
     election_borda = elections.Borda(pp, 1, score_vector=None)
-    election_irv = elections.STV(pp, fractional_transfer, 1)
-    election_plurality = elections.Plurality(pp, seats=1, ballot_ties=False)
-    election_seq = elections.SequentialRCV(pp, seats=1)
+    election_irv = elections.STV(pp)
+    election_plurality = elections.Plurality(pp)
+    election_seq = elections.SequentialRCV(pp)
     # election_sntv = elections.SNTV(pp, seats=1)
 
-    outcome_borda = election_borda.run_election().winners()
-    outcome_irv = election_irv.run_election().winners()
-    outcome_plurality = election_plurality.run_election().winners()
-    outcome_seq = election_seq.run_election().winners()
+    outcome_borda = election_borda.get_elected()
+    outcome_irv = election_irv.get_elected()
+    outcome_plurality = election_plurality.get_elected()
+    outcome_seq = election_seq.get_elected()
     # outcome_sntv = election_sntv.run_election().get_all_winners()
 
     print(outcome_borda)
```

Comment:
Why do the commented lines and print statements exist in this?

Reply:
No idea.